index.js

```diff
@@ -22,7 +26,14 @@ function getPercentageScore(score) {}
  * @param {object} score2 - An object representing another score of an assignment. Has the same shape as `score1`.
  * @returns {number} A rounded integer, representing the highest percentage scored among the two given scores.
  */
-function getHighestOfTwoScores(score1, score2) {}
+function getHighestOfTwoScores(score1, score2) {
+let result1 =  Math.round(score1.received / score1.max * 100)
+ let result2 = Math.round(score2.received / score2.max * 100)
```

Comment:
You could use the previous function, here. Try it!

Reply:
I did and It worked. I wonder did you mean something like this?
  
 let result1 =  Math.round(score1.received / score1.max * 100)
 let result2 = Math.round(score2.received / score2.max * 100)
 highestScore = getPercentageScore(result1, result2);
 if (result1 > result2 ) {
  return result1
}
else {
  return result2
}

}

I don't know how to share this in the form of a code block in this text box.

Reply:
Hey Tafari!

You've got the right idea, calling `getPercentageScore` and using its return value. But what parameters does it take in? What does it return? Once you've got that figured out, try using the function to set your `result1` and `result2` variables.

Reply:
Ok....how about this?


 result1 =  getPercentageScore(score1)
 result2 = getPercentageScore(score2)
if (result1 > result2) {
  return result1
}
else {
  return result2
}

}

Reply:
Great job! This is very DRY code ("Don't Repeat Yourself"). Instead of writing the logic for figuring out a percentage 3 times, you've only got it once, now. This has many advantages (feel free to research why people prefer DRY code!).

The last improvement you could make is to use your `Math.max` code here instead of the `if`/`else`.

Well done.